CMakeLists.txt

```diff
@@ -173,7 +167,7 @@ if(QUIC_TLS STREQUAL "openssl")
         OUTPUT ${QUIC_BUILD_DIR}/openssl/include
         OUTPUT ${QUIC_BUILD_DIR}/openssl/lib/libcrypto.a
         OUTPUT ${QUIC_BUILD_DIR}/openssl/lib/libssl.a
-        COMMAND ./Configure linux-x86_64 enable-tls1_3 --prefix=${QUIC_BUILD_DIR}/openssl
+        COMMAND ./config enable-tls1_3 --prefix=${QUIC_BUILD_DIR}/openssl
```

Comment:
I've tried this change before. Azure Pipelines build is going to fail, because it doesn't recognize the Azure Pipelines Windows image.

Reply:
```
[ 34%] Generating openssl/include, openssl/lib/libcrypto.a, openssl/lib/libssl.a
Operating system: x86_64-whatever-build
This system (build) is not supported. See file INSTALL for details.
CMakeFiles/OpenSSL.dir/build.make:79: recipe for target 'openssl/include' failed
```

Reply:
Don't we not currently support openssl on windows? Shoud we just be disabling it on Windows? We're going to have to special case Windows anyway.

Reply:
Err, I meant the Azure Pipelines **Linux** image, not Windows.

Reply:
I'll look into getting it on Windows to satisfy the issue you created a bit ago.

Reply:
https://developercommunity.visualstudio.com/content/problem/602584/openssl-build-error-when-using-pipelines.html